THIRD_PARTY_LICENSES/Cephes-README

```diff
@@ -0,0 +1,17 @@
+Cephes Math Library Release 2.8:  June, 2000
```

Comment:
This should copy over the extra note about the license:

Additional copyright information:
Code in this package is adapted from the Cephes library (http://www.netlib.org/cephes/).
" There is no explicit licence on Netlib, but the author has agreed to a BSD release.
See https://github.com/deepmind/torch-cephes/blob/master/LICENSE.txt and
https://lists.debian.org/debian-legal/2004/12/msg00295.html "

Reply:
Yeah, I was looking in to that last night. It's difficult, but I think this probably should be added too.

Reply:
I had a look at the debian-legal list comments and was not comfortable with adding anything more than what I have because of that; note that debian does not include cephes in the debs repo, possibly because of this.

I'd be happy to revisit this, but probably after having written to him with a specific text for a license file for him to approve.

Reply:
I have written to Steve Moshier to ask for his approval to include a BSD 3-clause license with his copyright.

Reply:
Done